pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix

```
@@ -9,6 +9,14 @@ trivialBuild rec {
     sha256 = "sha256-vJ+UlPMIuZ02da9R67wIq2dVaWElu/sYmWx2KgBQ9B8=";
   };
 
+  commit = "not-used-but-has-to-be-set";
```

```suggestion
  commit = "ef0991c2f0c385df5bbefbb2405f7873055556bc";
```

From their site:

https://gitweb.gentoo.org/proj/ebuild-mode.git/commit/?h=ebuild-mode-1.65

---

Why bother?  It builds fine. :)

Is [r-ryantm](https://github.com/r-ryantm) able to update `commit`?

---

It's possible with an updater script.

---

If this really builds fine, then `commit` is dead code and should be removed from all other packages using `melpaBuild`.

---

I will check the upstream MELPA build script and see if there is something we can do in a few days.

---

For now I will merge this, even not liking it.